.github/workflows/test.yml

```diff
@@ -14,16 +14,19 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        django-version: ["32", "40", "41"]
```

Comment:
what about Django 4.2?

Reply:
Not supported yet so I wanted to keep this PR slim and not introduce in the worst case a failing pipeline. Once this is merged, I'd enable it and fix possible issues.

Reply:
@nemesifier I found a bug in my setup. I fixed it, now I'm sure that all tests are actually run against the dedicated Django version.

Reply:
That's great thanks, I didn't have time to review closely yet. Can you please add Django 4.2?

Reply:
Done 👌

Reply:
@nemesifier Do you need anything else in this PR?